src/raw/lined/helix.svg

```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" width="24" height="24"
+  stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10">
+  <path
+    d="M7.5 3.5a8.492 8.492 0 001.4 5.3M16 3.5c0 2.6-1.6 5.5-3.8 7.7S7.1 15 7.1 20M14.3 14.7a9.654 9.654 0 011.7 5.8" />
+</svg>
```

Comment:
@dexterca this is the only one I changed the name on.  In chroma, this is `helix-alt`, since we have a `helix`.  But our `helix` in chromicons is named `dna`.  Thoughts?

Should we:

- name it `helix-alt`
- leave this as `helix`
- name it `dna-alt`
- name it `dna-2`

Reply:
I think `dna-2` is the right choice in this stage of the game. Wish I would have caught `dna` before because I prefer `helix`. I don't want to cause a breaking change at this point.

Reply:
Up to you on breaking changes - I don't mind since it's only affecting PHC and I need to do an update in there anyway.

Reply:
ok, lets do `helix` and `helix-2`! thx!

Reply:
💥